rythm.js

```diff
@@ -12,7 +12,7 @@ var classCallCheck = function (instance, Constructor) {
 
 var createClass = function () {
   function defineProperties(target, props) {
-    for (var i = 0; i < props.length; i++) {
+    for (var i = 0, len = props.length; i < len; i++) {
```

Comment:
Could you explain ?

Reply:
> for (var i = 0; i < props.length; i++)
if props.length is 1000, the props.length will invoke 1000 times.
> for (var i = 0, len = props.length; i < len; i++)
this way props.length just invoke one time.

Reply:
[For-loop performance: storing array length in a variable](https://stackoverflow.com/questions/17989270/for-loop-performance-storing-array-length-in-a-variable#answer-17995000)

Maybe i am wrong here. 

Like this:
> document.getElementsByTagName('a').length

Cache the length may be better, but other situations i am not sure.

Reply:
I made a benchmark. And they're so close that the result is not consistent. I heard that it could be a V8 optimisation. Since we have no clues i prefer it to stay as it is. If we run into performance issues we will investigate how to improve, but right now this is not necessary.

Reply:
Thanks a lot, I reall appreciate your serious attitude and strong executive force. Good job.

Reply:
Do I need to make another commit to change it back?

Reply:
It would be perfect :)

Reply:
I changed it back, Thanks.